proto/internal/dispatch/v1/dispatch.proto

```diff
@@ -161,7 +161,8 @@ message DispatchLookupSubjectsRequest {
 
 message FoundSubject {
   string subject_id = 1;
-  repeated string excluded_subject_ids = 2;
+  CaveatExpression conditional_expression = 2;
```

Comment:
It only really matters if we intend back compat and to do so we'd have to fill in the excluded IDs as well. We could do that, but it would require keeping around the extra logic